docs/sources/datasources/tempo/_index.md

```diff
@@ -57,51 +57,52 @@ The **Trace to logs** setting configures the [trace to logs feature]({{< relref
-| \_\_span.tags      | Namespace for the tags in the span. To access a specific tag named "version" you would use `${__span.tags.version}`. In case the tag contains dot you have to access it as `${__span.tags["http.status"]}`.                                                                                                                                                        |
-| \_\_trace.traceId  | The ID of the trace.                                                                                                                                                                                                                                                                                                                                               |
-| \_\_trace.duration | The duration of the trace.                                                                                                                                                                                                                                                                                                                                         |
-| \_\_trace.name     | The name of the trace.                                                                                                                                                                                                                                                                                                                                             |
+| Variable name          | Description                                                                                                                                                                                                                                                                                                                                                        |
+| ---------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| **\_\_tags**           | This variable is special because it uses the tag mapping in from the UI to create a label matcher string in the specific data source syntax. It uses only tags that are present in the span so the link will still be created even if only one of those tags is present in the span. You can use this if not all the tags are required for the query to be useful. |
```

Comment:
In the second sentence, it says: "...so the link will still be created even if only one of those tags is present in the span." I am not sure which (or what) link is created. Will it be clear to a reader what is meant? Maybe there is a way to reword this text so it's clearer, as well. 
```suggestion
| **\_\_tags**           | This variable uses the tag mapping in from the UI to create a label matcher string in the specific data source syntax. The variable only tags that are present in the span. The link is still created even if only one of those tags is present in the span. You can use this if all tags are not required for the query to be useful. |
```